src/PrestaShopBundle/Resources/views/Admin/WebProfiler/config.html.twig

```diff
@@ -43,7 +43,7 @@
     {% endif %}
 
     {% set icon %}
-        <span class="sf-toolbar-label"><img src="{{ asset('themes/new-theme/scss/img/glyph.png') }}" /></span>
+        <span class="sf-toolbar-label"><img src="{{ asset('themes/new-theme/public/glyph.png') }}" /></span>
```

Comment:
The directory `public` is removed each time you compile assets. you must put this file in the `themes/new-theme/img` directory

Reply:
Forget to say, you can remove the file in the scss/img if it's not used in the css :D